### ISSUE TITLE
Bot does not accept match

### ISSUE THREAD

**Issue by MekaIis** [TeamFightTacticsBots/Alune#94]
Bot works, but when it clicks "play" in cmd bot is only clicking button play, even if he should accept.

**Comment by akshualy** [comment]
Check your logs in `alune-output/logs`. Does it lock into queue logic and does it say "queue accepted"? If not, is your TFT in English?

**Comment by JokoKenguru** [comment]
but you said that he didn't focus on languages..

**Comment by akshualy** [comment]
> but you said that he didn't focus on languages..

Yes, I *try* to do that, but some languages have particularily high or wide characters, characters for entire words, or longer words for simpler things. That changes by what we can or should recognize a button being present.

**Comment by akshualy** [comment]
Closing as there is no response if this was the solution.
